### PR TITLE
Proof-of-concept combined landsat-sentinel-2 layer.

### PR DESCRIPTION
**Pull request by SpacemanPaul**
Proof-of-concept combined landsat-sentinel-2 layer.  

Visible bands only, no masking.  Dev environment only.

### CODE REVIEW

**Comment by robbibt** [comment]
This looks awesome, thanks @SpacemanPaul! Is there any way to preview the web services before this is merged, or is it easier to just merge it then do a more detailed review once it's live in dev OWS?

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+bands_ls_s2_shared = {
+    "nbart_blue": ["nbart_blue", "blue"],
+    "nbart_green": ["nbart_green", "green"],
+    "nbart_red": ["nbart_red", "red"],
+}
+
+combined_layer = {
+    "title": "DEA Surface Reflectance (Combined Sentinel-2/Landsat)",
+    "name": "s2_ls_combined",
+    "abstract": """Geoscience Australia Landsat Analysis Ready Data Collection 3
```
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:15]: Ah, a title change needed here as this contains both S2 and Landsat

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# bands definition
+bands_ls_s2_shared = {
+    "nbart_blue": ["nbart_blue", "blue"],
+    "nbart_green": ["nbart_green", "green"],
+    "nbart_red": ["nbart_red", "red"],
+}
+
+combined_layer = {
+    "title": "DEA Surface Reflectance (Combined Sentinel-2/Landsat)",
```
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:13]: Can we just call this "DEA Surface Reflectance (Landsat and Sentinel-2)" to match the existing name?

**Review by robbibt — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py (hunk 1 above)
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:15]:
```suggestion
    "abstract": """Geoscience Australia Landsat and Sentinel-2 Analysis Ready Data Collection 3
```

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py (hunk 2 above)
Comment (robbibt) [inline dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:13]:
```suggestion
    "title": "DEA Surface Reflectance (Landsat and Sentinel-2)",
```

**Comment by SpacemanPaul** [comment]
> This looks awesome, thanks @SpacemanPaul! Is there any way to preview the web services before this is merged, or is it easier to just merge it then do a more detailed review once it's live in dev OWS?

Yes easiest to merge, then test in dev.  That's kind of what dev is for.

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py (hunk 1 above)
Comment (SpacemanPaul) [reply dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:15]: `s2_ls_combined`:    s2=Sentinel-2  ls=landsat.

Are you requesting a change here?

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py (hunk 2 above)
Comment (SpacemanPaul) [reply dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py:13]: Yes, of course.   Shall do.

**Review by robbibt — APPROVED** [review]
Looks great to me!

File: dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py (hunk 1 above)
Comment (robbibt) [reply dev/services/wms/ows_refactored/baseline_satellite_data/landsat_s2_combined/ows_c3_cfg.py]: Nah, I think that line is fine, it was just the first line of the abstrac I was referring to

**Pull request merged**